OneCommunityGlobal/HGNRest
# XiaoW_new timer refactor

# Description
This is the refactored new timer frontend, and the related files were based on PR [#450](https://github.com/OneCommunityGlobal/HGNRest/pull/450#issue-1811283656). 
The new timer is based on Websocket connection communication and it is a count down timer instead of stop watch timer like the old timer.

## Related PRS (if any):
This frontend PR is related to the [#1282](https://github.com/OneCommunityGlobal/HighestGoodNetworkApp/pull/1282#issue-1886998437) frontend PR.
To test this frontend PR you need to checkout the [#1282](https://github.com/OneCommunityGlobal/HighestGoodNetworkApp/pull/1282#issue-1886998437) frontend PR.

## How to test:
1. check into current branch
2. check into frontend branch
3. do `npm install` and `npm run dev` to run this PR locally
4. follow the instructions on the frontend branch

## Note:
1. `npm run dev` will automatically build and start the server, and rebuild and restart when any changes is saved in `/src` directory, so there is no need to run `npm run build` along with `npm start` any more.
